litmus-portal/graphql-server/pkg/database/mongodb/usermanagement/operations.go

```diff
@@ -40,6 +41,70 @@ func GetUserByUserName(ctx context.Context, username string) (*User, error) {
 	return user, err
 }
 
+// DeactivateUser updates the details of user in both user and project collections
+func UpdateUserState(ctx context.Context, user User) error {
```

Comment:
```suggestion
func DeactivateUser(ctx context.Context, user User) error {
```

Reply:
This can be used to deactivate or reactivate user's state, that's why a generic name

Reply:
Then update the comment